scripts/typedef.js

```diff
@@ -34,6 +34,21 @@
  * @property {Address} bracketAddress address of bracket into which to deposit
  */
 
+/**
+ * Example:
+ * {
+ *   amount: 100,
+ *   tokenAddress: 0x0000000000000000000000000000000000000000,
+ *   toAddress: 0x0000000000000000000000000000000000000001
+ * }
+ *
+ * @typedef Transfer
+ * @type {object}
+ * @property {number|string|BN} amount integer denoting amount to be transfered
```

Comment:
Only integers? It does not match the example where you have decimals.

Reply:
```suggestion
 * @property {number|string|BN} amount value denoting amount to be transferred
```